lib/api/1.1/northbound/profiles.js

```diff
@@ -2,9 +2,9 @@
 
 'use strict';
 
-var di = require('di'),
-    express = require('express'),
-    parser = require('body-parser');
+var di = require('di');
+var express = require('express');
+var ambiguousParser = require('./../../../fittings/ambiguous_text_parser');
```

Comment:
I'm almost inclined to suggest just embedding the middleware function into both profiles.js and templates.js, since it's dead simple and only used in two places, but I don't know how that will play with the 2.0 API transition either.